cob-base-reasoned.owl

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0"?>
-<rdf:RDF xmlns="http://purl.obolibrary.org/obo/cob/cob-base-reasoned.owl#"
-     xml:base="http://purl.obolibrary.org/obo/cob/cob-base-reasoned.owl"
+<rdf:RDF xmlns="http://purl.obolibrary.org/obo/cob/cob-edit.owl#"
+     xml:base="http://purl.obolibrary.org/obo/cob/cob-edit.owl"
```

Comment:
I have no idea how that change came about.

Reply:
This is probably an unfortunate side effect of our prefix roundtrip fix in ROBOT, from the automatically generated default `:` prefix in the first ontology loaded.